pkg/operator/k8sutil/k8sutil.go

```diff
@@ -206,3 +207,14 @@ func StartOperatorSettingsWatch(context *clusterd.Context, operatorNamespace, op
 		})
 	go cacheController.Run(stopCh)
 }
+
+func UsePDBV1Version(Clientset kubernetes.Interface) (bool, error) {
```

Comment:
What about calling this `UsePDBBetaV1Version` and default everything to v1. So moving forward, when we support a Kubernetes version that only has v1 we can simply remove this function and the check without changing too much logic.

Reply:
Done